src/app/api/organization/[id]/route.ts

```diff
@@ -0,0 +1,23 @@
+import { NextResponse, NextRequest } from "next/server";
+import connectDB from "../../../../database/db";
+import Organization from "../../../../database/organizationSchema";
+
+// "@/app" path is not working for some reason
```

Comment:
small note, should be able to use `import Organization from "@/database/organizationSchema";`